snuba/datasets/configuration/generic_metrics/storages/counters.yaml

```diff
@@ -77,6 +77,26 @@ allocation_policies:
         is_enforced: 1
         throttled_thread_number: 1
         org_limit_bytes_scanned: 100000
+  - name: CrossOrgQueryAllocationPolicy
+    args:
+      required_tenant_types:
+        - referrer
+      default_config_overrides:
+        is_enforced: 0
+        is_active: 0
+      cross_org_referrer_limits:
+        dynamic_sampling.counters.get_org_transaction_volumes:
```

Comment:
I understand wanting to use these fields as specific overrides, but why not use the `cross_org_query` flag to determine if the referrer should be included or not? 

In other words, if a referrer comes in with `cross_org_query=1`, automatically use the cross org policy with `max_threads=4` and `concurrent_limit=10`?